### PR TITLE
Fix copy/paste that was never updated

### PR DESCRIPTION
**Pull request by trevordmiller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -884,4 +884,4 @@ Currently mobile is not supported. However, there are plans to add touch support
 
 ## Author / maintainer
 
-Alex Reardon - [@alexandereardon](https://twitter.com/alexandereardon) - areardon@atlassian.com
\ No newline at end of file
+Alex Reardon - [@alexandereardon](https://twitter.com/alexandereardon) - areardon@atlassian.com
```
Comment (alexreardon) [inline README.md]: not sure what the change is on the last line

File: README.md [hunk 2]
```diff
@@ -562,7 +562,7 @@ import { Draggable } from 'react-beautiful-dnd';
 
 - `draggableId`: A *required* `DraggableId(string)` that uniquely identifies the `Draggable` for the application. Please do not change this prop - especially during a drag.
 - `type`: An *optional* type (`TypeId(string)`) of the `Draggable`. This is used to control what `Droppable`s the `Draggable` is permitted to drop on. `Draggable`s can only drop on `Droppable`s that share the same `type`. If no `type` is provided, it will be set to `'DEFAULT'`. Currently the `type` of a `Draggable` **must be** the same as its container `Droppable`. This restriction might be loosened in the future if there is a valid use case.
-- `isDragDisabled`: An *optional* flag to control whether or not dropping is currently allowed on the `Droppable`. You can use this to implement your own conditional dropping logic. It will default to `false`.
+- `isDragDisabled`: An *optional* flag to control whether or not dragging is currently allowed on the `Draggable`. You can use this to implement your own conditional dragging logic. It will default to `false`.
```
Comment (alexreardon) [inline README.md]: ❤️

File: README.md (hunk 1 above)
Comment (trevordmiller) [inline README.md]: I don't know either - I've never seen this with any of the other pull requests I submit. I'll close this and you can do it directly. Sorry for the hassle.

Comment (alexreardon) [inline README.md]: Nah all good - sorry for the trouble too! Please do not hesitate to submit other PR's going forward 👍

**Pull request closed without merging**